release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -466,19 +466,115 @@ With this update, the default credentials request for AWS has been modified to a
+
+* Renaming, or otherwise changing, the default storage class
+
+* Enforcing static provisioning by disabling dynamic provisioning.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc[Managing the default storage class].
+
+[id="ocp-4-13-storage-retroactive-default-sc-assignment"]
+==== Retroactive default StorageClass assignment (Technology Preview)
+Previously, if there was no default storage class, persistent volumes claims (PVCs) that were created that requested the default storage class remained stranded in the pending state indefinitely, unless you manually delete and recreate them. {product-title} can now retroactively assign a default storage class to these PVCs, so that they do not remain in the pending state. With this feature enabled, after a default storage class is created, or one of the existing storage classes is declared the default, these previously stranded PVCs are assigned the default storage class.
+
+This feature is supported with Technology Preview status.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc[Managing the default storage class].
+
+[id="ocp-4-13-storage-csi-inline-ephemeral-vols-GA"]
+==== CSI inline ephemeral volumes is generally available
+Container Storage Interface (CSI) inline ephemeral volumes were introduced in {product-title} 4.5 as a Technology Preview feature, which allows you to define a pod spec that creates inline ephemeral volumes when a pod is deployed and delete them when a pod is destroyed. This feature is now generally available.
+
+This feature is only available with supported Container Storage Interface (CSI) drivers.
+
+This feature also includes the CSI Volume Admission plug-in, which provides a mechanism where the use of an individual CSI driver capable of provisioning CSI ephemeral volumes can be restricted on pod admission. Administrators or distributions can add a `csi-ephemeral-volume-profile` label to a `CSIDriver` object, and the label is then inspected by the Admission plug-in and used in enforcement, warning, and audit decisions.
+
+For more information, see xref:../storage/container_storage_interface/ephemeral-storage-csi-inline.adoc[CSI inline ephemeral volumes].
+
+[id="ocp-4-13-storage-csi-migration-azure-file-GA"]
+==== Automatic CSI migration for Microsoft Azure File is generally available
+Starting with {product-title} 4.8, automatic migration for in-tree volume plugins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for Azure File was provided in this feature in {product-title} 4.10. {product-title} 4.13 now supports automatic migration for Azure File as generally available. CSI migration for Azure File is now enabled by default and requires no action by an administrator.
```

Comment:
WDYM by 

> Support for Azure File was provided in this feature in {product-title} 4.10

Azure file became GA in 4.11 see STOR-736. I think the sentence is confusing maybe we can make it more clear?